src/main/kotlin/com/manimdsl/runtime/ASTExecutor.kt

```diff
@@ -209,6 +210,26 @@ class VirtualMachine(
             is ForStatementNode -> executeForStatement(statement)
             is LoopStatementNode -> executeLoopStatement(statement)
             is InternalArrayMethodCallNode -> executeInternalArrayMethodCall(statement)
+            is StartSpeedChangeNode -> {
```

Comment:
Could we add a runtime check to see if the specified speed is > 0? Doesn't really make sense to have non-positive speeds I think